src/pages/home/report/comment/TextWithEmojiFragment/index.ios.tsx

```diff
@@ -3,27 +3,28 @@ import {View} from 'react-native';
 import Text from '@components/Text';
 import useThemeStyles from '@hooks/useThemeStyles';
 import convertToLTR from '@libs/convertToLTR';
-import * as EmojiUtils from '@libs/EmojiUtils';
+import {containsCustomEmoji, splitTextWithEmojis} from '@libs/EmojiUtils';
 import type TextWithEmojiFragmentProps from './types';
 
 function TextWithEmojiFragment({message = '', style}: TextWithEmojiFragmentProps) {
     const styles = useThemeStyles();
-    const processedTextArray = useMemo(() => EmojiUtils.splitTextWithEmojis(message), [message]);
+    const processedTextArray = useMemo(() => splitTextWithEmojis(message), [message]);
 
     return (
         <Text style={style}>
-            {processedTextArray.map(({text, isEmoji}, index) =>
-                isEmoji ? (
+            {processedTextArray.map(({text, isEmoji}, index) => {
+                const textContainsCustomEmoji = containsCustomEmoji(text);
+                return isEmoji ? (
                     <View
                         // eslint-disable-next-line react/no-array-index-key
                         key={index}
                     >
-                        <Text style={styles.emojisWithTextFontSizeAligned}>{text}</Text>
+                        <Text style={[styles.emojisWithTextFontSizeAligned, textContainsCustomEmoji && styles.customEmojiFontAlignment]}>{text}</Text>
```

Comment:
```suggestion
                        <Text style={[styles.emojisWithTextFontSizeAligned, containsCustomEmoji(text) && styles.customEmojiFontAlignment]}>{text}</Text>
```

This is a nitpick, but we don't need to make `containsCustomEmoji` check if `isEmoji=false`